docs/git-cola.rst

```diff
@@ -762,6 +762,12 @@ Defaults to `true`.  This setting is configured using the `Preferences`
 dialog, but it can be toggled for one-off usage using the commit message
 editor's options sub-menu.
 
+cola.logdate
+--------------
+Set the default date-time mode for the DAG display. Setting a value for
+cola.logdate is similar to using git log's --date option.
+See `git log(1) <https://www.git-scm.com/docs/git-log>`_ for details.
+
```

Comment:
We should use this URL (with the anchor value) so that we link directly to the `--date=<format>`  docs.

`https://git-scm.com/docs/git-log#Documentation/git-log.txt---dateltformatgt`

https://git-scm.com/docs/git-log#Documentation/git-log.txt---dateltformatgt


Also, instead of "Similar", maybe it should say, "This value is passed to `git log --date=<format>`"?